ingest.py

```diff
@@ -14,23 +30,47 @@
 load_dotenv()
 
 
+# Map file extensions to document loaders and their arguments
+LOADER_MAPPING = {
+    ".csv": (CSVLoader, {}),
+    # ".docx": (Docx2txtLoader, {}),
+    ".docx": (UnstructuredWordDocumentLoader, {}),
+    ".enex": (EverNoteLoader, {}),
+    ".eml": (UnstructuredEmailLoader, {}),
+    ".epub": (UnstructuredEPubLoader, {}),
+    ".gcsdir": (GCSDirectoryLoader, {}),
+    ".gcsfile": (GCSFileLoader, {}),
```

Comment:
According to the documentation, this loader works differently:

`loader = GCSFileLoader(project_name="aist", bucket="testing-hwc", blob="fake.docx")`

Also requires a new dependency: 
pip install google-cloud-storage

Same applies for `GCSDirectoryLoader`

Reply:
I removed those - I agree that those are not ok.